packages/flutter/lib/src/material/data_table.dart

```diff
@@ -163,6 +165,12 @@ class DataRow {
   /// Otherwise, the checkbox, if present, will not be checked.
   final bool selected;
 
+  /// Whether the row can be selected by using gestures such as
+  /// tap or long press.
+  ///
+  /// This value is true by default.
+  final bool selectableOnGestures;
```

Comment:
Is there reference to this in spec? I imagine this would be a mobile/tablet behavior only.

Reply:
I didn't find any specs, this is in response to the users in https://github.com/flutter/flutter/issues/70755.

Reply:
> I imagine this would be a mobile/tablet behavior only.

This is for all platforms where we don't expect users to tap the row when onTap or long-press are not implemented but only `onSelectChanged`, which makes the row tappable with inkwell effect and also affects accessibility (talkback says "double-tap to activate").  

`onTap == null` and `onSelectChanged != null` makes wrap label with InkWell.

Reply:
Gotcha. Can you please add an explanation for how this interacts with `onSelectChanged` and `onRowLongPress`?

Reply:
Add an explanation, please take a look.